xml/dhcp_vendor_class.xml

```diff
@@ -578,6 +578,15 @@
     <param pos="0" name="os.certainty" value="0.8"/>
   </fingerprint>
 
+  <fingerprint pattern="^Avaya E129 dslforum.org$">
```

Comment:
The `dslforum.org` part stands out as a bit odd since that is a broadband discussion forum. Do you have any additional information to clarify the relationship between these?

Reply:
I see one Aruba fingerprint that includes this string as well. I wasn't involved in the review of that PR so I'm curious.

Reply:
Disregard that comment, I was thinking of DSL Reports! :)